journal/modernized/15-chapter-14.adoc

```diff
@@ -354,7 +372,9 @@ and accept a large share yourself, from your sincere friend,
 Deborah Darby
 
 [.signed-section-context-close]
-Godalming, 5th month 30th, 1802.
+Godalming, 5th month 30th, 1802
+
+--.
```

Comment:
Looks like a period got stuck after the double dash here `--.` that needs to be removed.